fiftyone/operators/executor.py

```diff
@@ -202,25 +295,11 @@ def selected(self):
         """The list of selected sample IDs or an empty list."""
         return self.request_params.get("selected", [])
 
-    @property
```

Comment:
Inadvertently deleted? `ctx.selected_labels` was a feature that @ritch added recently. Maybe its a release branch vs develop issue?